boards/shields/nrf700x_nrf54h20dk/Kconfig.shield

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+DT_COMPAT_NORDIC_NRF700X_SPI := nordic,nrf700x-spi
+
+config SHIELD_NRF700X_NRF54H20DK
+	def_bool $(shields_list_contains,nrf700x_nrf54h20dk)
+
+config NRF7002_ON_SPI
+	def_bool $(dt_compat_enabled,$(DT_COMPAT_NORDIC_NRF700X_SPI))
+	depends on SHIELD_NRF700X_NRF54H20DK
+
+config BOARD_NRF7002
+	bool
+	default y if SHIELD_NRF700X_NRF54H20DK
```

Comment:
?

Reply:
These are generic BOARD defines used elsewhere in the code.

Reply:
I feel this is wrong @tejlmand. What does it mean "generic board defines"?

Reply:
This define is to identify a common chip type (independent to board name) but I see that it's unused, so, will remove this.